entwatch/ze_einstein_v7.cfg

```diff
@@ -1,46 +1,6 @@
 "entities"
-{
+{	
 	"0"
-	{
-		"name"              "Wall"
-		"shortname"         "Wall"
-		"color"             "{green}"
-		"buttonclass"       "func_button"
-		"filtername"        ""
-		"hasfiltername"     "false"
-		"blockpickup"       "false"
-		"allowtransfer"     "true"
-		"forcedrop"         "true"
-		"chat"              "true"
-		"hud"               "true"
-		"hammerid"          "1981"
-		"mode"              "3"
-		"maxuses"           "1"
-		"cooldown"          "0"
-		"maxamount"         "20"
-	}
-	
-	"1"
-	{
-		"name"              "Sexy Wall"
-		"shortname"         "Sexy Wall"
-		"color"             "{green}"
-		"buttonclass"       "func_button"
-		"filtername"        ""
-		"hasfiltername"     "false"
-		"blockpickup"       "false"
-		"allowtransfer"     "true"
-		"forcedrop"         "true"
-		"chat"              "true"
-		"hud"               "true"
-		"hammerid"          "319475"
-		"mode"              "3"
-		"maxuses"           "1"
-		"cooldown"          "0"
-		"maxamount"         "4"
-	}
-	
-	"2"

```

Comment:
Reasoning for this as well? If it's spam, you can just toggle chat/hud off. Message will still show in chat when they get used (not nothing else).

Reply:
The problem is that it doesn't show a message when they are used and also won't change the uses from [0/1] to [D]

Reply:
Tried multiple things to fix this but I got no idea at this point. Nothing in the system is sticking out to me vs other maps where use tracking does work.